__tests__/account.test.ts

```diff
@@ -48,20 +45,18 @@ describe('deploy and test Wallet', () => {
     const dappResponse = await provider.deployContract({
       contract: compiledTestDapp,
     });
-    dapp = new Contract(compiledTestDapp.abi, dappResponse.address!, provider);
-    expect(dappResponse.code).toBe('TRANSACTION_RECEIVED');
+    dapp = new Contract(compiledTestDapp.abi, dappResponse.contract_address!, provider);
 
     await provider.waitForTransaction(dappResponse.transaction_hash);
   });
 
   test('estimate fee', async () => {
-    const { amount, unit } = await account.estimateFee({
+    const { overall_fee } = await account.estimateFee({
```

Comment:
can we get this change into v3 too?
I think v0.9.1 should still be supported by both starknet.js versions

Reply:
#248 should cover that